.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Test tephi
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
This repo currently doesn't use a `main` branch (still called `master`)

Reply:
That explains why I couldn't get it to test on my fork!